public/js/clients/firefox/commands.js

```diff
@@ -98,6 +96,16 @@ function getProperties(grip) {
   return objClient.getPrototypeAndProperties();
 }
 
+function prettyPrint(sourceId, indentSize) {
+  const sourceClient = threadClient.source({ actor: sourceId });
+  return sourceClient.prettyPrint(indentSize);
+}
+
+function disablePrettyPrint(sourceId) {
+  const sourceClient = threadClient.source({ actor: sourceId });
+  return sourceClient.disablePrettyPrint();
+}
```

Comment:
ahh - yeah, i was thinking something like `togglePrettyPrint(wantPretty)`. but it doesn't matter